components/Procedure/src/index.scss

```diff
@@ -109,28 +108,32 @@
 }
 
 .denhaag-procedure .denhaag-link--external {
-  --denhaag-link-external-display: var(--denhaag-procedure-link-external-display);
+  --denhaag-link-external-display: var(--denhaag-procedure-link-external-display, inline-flex);
 }
 
 .denhaag-procedure .denhaag-link--external .denhaag-link__icon {
```

Comment:
Dit stukje vindt ik heel confusing, align self van de icon wordt gezet in CSS variabel waarna een nieuwe property een compleet andere CSS variabel gebruikt?